src/dashboards/videos/details.jsonnet

```
@@ -392,9 +374,7 @@ dashboard.new(
         timeField: 'timestamp',
       },
     ],
-    type: 'grafana-groupedbarchart-panel',
-    valueName: 'current',
-    width: 800,
+    type: 'potsie-stackedbarchart-panel',
   },
-  gridPos={ x: 0, y: 17, w: 24, h: 16 }
+  gridPos={ x: 0, y: 17, w: 24, h: 9 }
```

I would exchange its position with row of verbs and video event distribution as it is THE attraction of this panel row
WDYT?
```suggestion
  gridPos={ x: 0, y: 8, w: 24, h: 9 }
```